### PR TITLE
chore(list): add timeout for calcContentHeight function in componentD…

### LINKED ISSUE #672: [bal-list][accordion group] if the accordions are open from the beginning, their height is not calculated well
Screenshot 2022-10-03 at 17 52 04

Steps to reproduce:
1. go to bal-list.stories.ts
2. add the following code snippet
 ``` 
export const OpenAccordionGroupList = args => ({
  components: { ...component.components, BalIcon, BalListItemAccordionHead, BalListItemAccordionBody },
  setup: () => ({ args }),
  template: `<div class="columns">
  <div class="column">
  <bal-list v-bind="args">
  <bal-list-item accordion>
    <bal-list-item-accordion-head accordion-open>
      <bal-list-item-icon>
        <bal-icon name="account"></bal-icon>
      </bal-list-item-icon>
      <bal-list-item-content>
        <bal-list-item-title>Accordion List Item</bal-list-item-title>
      </bal-list-item-content>
    </bal-list-item-accordion-head>
    <bal-list-item-accordion-body accordion-group="bubu" open>
      <p class="pb-4">
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </p>
    </bal-list-item-accordion-body>
  </bal-list-item>
</bal-list>
  </div>
  <div class="column">
  <bal-list v-bind="args">
  <bal-list-item accordion>
    <bal-list-item-accordion-head accordion-open>
      <bal-list-item-icon>
        <bal-icon name="account"></bal-icon>
      </bal-list-item-icon>
      <bal-list-item-content>
        <bal-list-item-title>Accordion List Item</bal-list-item-title>
      </bal-list-item-content>
    </bal-list-item-accordion-head>
    <bal-list-item-accordion-body accordion-group="bubu" open>
      <p class="pb-4">
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </p>
    </bal-list-item-accordion-body>
  </bal-list-item>
</bal-list>
  </div>
  </div>`,
})
OpenAccordionGroupList.args = {
  border: true,
}
OpenAccordionGroupList.parameters = {
  ...component.sourceCode(OpenAccordionGroupList),
  controls: { exclude: excludedControls },
}
```

3. see the story: [Open Accordion Group List](http://localhost:6006/?path=/story/components-list--open-accordion-group-list) (empty cache & hard reload)

### PR DESCRIPTION
**Pull request by mladenplaninicic**
fix the wrong height value when accordion body has open prop as true in the initial loading of the page

Closes #

{{short description}}

## Acceptance Criteria

- [ ] Design Review by @Gagne87, @clastzoo
- [x] Technical Review by @hirsch88, @mladenplaninicic, @nobilo
- [ ] [Style Guide](https://stenciljs.com/docs/style-guide) Review @hirsch88, @mladenplaninicic, @nobilo
- [ ] Visual test done by @hirsch88, @nobilo, @mladenplaninicic, @Gagne87, @clastzoo
- [ ] Functional test done by @hirsch88, @nobilo, @mladenplaninicic
- Browser Testing by @hirsch88, @nobilo, @mladenplaninicic, @Gagne87, @clastzoo
    - Desktop
        - [ ] Chrome
        - [ ] Edge
        - [ ] Safari
        - [ ] Firefox
    - Tablet
        - [ ] iPad (Landscape / Portrait)
    - Mobile
        - [ ] Safari iOS
        - [ ] Chrome Preview

### CODE REVIEW

**Review by nobilo — APPROVED** [review]

**Comment by madaluca** [comment]
tested it on my local and it works

File: packages/components/src/components/bal-list/bal-list-item-accordion-body/bal-list-item-accordion-body.tsx [hunk 1]
```diff
@@ -33,7 +33,9 @@ export class ListItemAccordionBody {
   }
 
   componentDidRender() {
-    this.calcContentHeight()
+    setTimeout(() => {
```
Comment (nobilo) [inline packages/components/src/components/bal-list/bal-list-item-accordion-body/bal-list-item-accordion-body.tsx:36]: maybe to move it in componentDidLoad :)

**Review by nobilo — APPROVED** [review]

File: packages/components/src/components/bal-list/bal-list-item-accordion-body/bal-list-item-accordion-body.tsx (hunk 1 above)
Comment (mladenplaninicic) [reply packages/components/src/components/bal-list/bal-list-item-accordion-body/bal-list-item-accordion-body.tsx:36]: moved

Comment (madaluca) [reply packages/components/src/components/bal-list/bal-list-item-accordion-body/bal-list-item-accordion-body.tsx:36]: it's not working anymore with the new modification
Screenshot 2022-10-04 at 17 22 28

**Review by madaluca — APPROVED** [review]

**Pull request merged**